libclc/test/math/fabs.cl

```diff
@@ -0,0 +1,32 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: %clang -target amdgcn-mesa-mesa3d -O1 -emit-llvm -S -o - --libclc-lib=tahiti-amdgcn-mesa-mesa3d -fno-builtin --no-offloadlib %s | FileCheck %s
```

Comment:
This should specify the -mcpu, and the driver should figure out the appropriate library. It shouldn't be given an explicit subtarget name

Reply:
Agreed. I'd like to do that as a follow-up to the aforementioned clang PR.

To be clear, do you still imagine a user specifying `--libclc-lib=amdgcn-mesa-mesa3d` and the driver infers the `tahiti` part from `-mcpu`? Or does the user just specify `--libclc-lib` with no value to signify they want libclc, and the driver uses both `-target` and `-mcpu` to come to the right library name?

Reply:
Doesn't clang manage this now?